src/foam/u2/stack/BreadcrumbView.js

```diff
@@ -29,27 +35,29 @@ foam.CLASS({
       this.SUPER();
       var self = this;
       this.addClass(this.myClass('display'));
-      var navStack = this.stack.stack_.slice(this.stack.navStackBottom, this.stack.pos);
-      var themeIcon = navStack.length == 1 ? 'back' : '';
-      navStack.map((v, i, _) =>{
-        var index = i + this.stack.navStackBottom;
-        var jumpAction  = self.Action.create({
-          name: 'back',
-          code: () => {
-            self.stack.jump(index, self);
+      if ( this.stack && this.stack.stack_ ) { 
```

Comment:
Not sure why git is reading this as a massive change but only this condition was added in this file